languages.toml

```diff
@@ -3810,3 +3810,17 @@ language-servers = ["circom-lsp"]
 [[grammar]]
 name   = "circom"
 source = { git = "https://github.com/Decurity/tree-sitter-circom", rev = "02150524228b1e6afef96949f2d6b7cc0aaf999e" }
+
+[[language]]
+name = "nginx"
+scope = "source.nginx"
+injection-regex = "nginx"
+file-types = ["conf"]
```

Comment:
This will conflict with `hocon`. We might need a glob pattern that would match the common places to put nginx files instead